crates/bevy_window/src/window.rs

```diff
@@ -32,6 +32,25 @@ impl Default for WindowId {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct WindowResizeConstraints {
+    pub min_width: f32,
+    pub min_height: f32,
+    pub max_width: f32,
+    pub max_height: f32,
+}
+
+impl Default for WindowResizeConstraints {
+    fn default() -> Self {
+        Self {
+            min_width: 360.,
+            min_height: 240.,
+            max_width: 23040.,
+            max_height: 8640.,
```

Comment:
I think there should be no default maximum size (so no `.with_max_inner_size` call by default) and a smaller default minimum size.

Reply:
Ok, I will remove max size. What minimum resolution do you propose for a window?

Reply:
Maybe 180x120?